c7n/resources/cw.py

```diff
@@ -466,6 +468,87 @@ def get_dimensions(self, resource):
         return [{'Name': 'LogGroupName', 'Value': resource['logGroupName']}]
 
 
+@resources.register('log-metric-filter')
```

Comment:
can we drop the -filter part of this in the resource name, feels a little confusing talking about the filter's filters. and in the end  it is just a metric from logs.